lib/network/modules/components/Node.js

```diff
@@ -716,6 +716,18 @@ class Node {
       options.mass = 1;
     }
   }
+
+  /**
+  * Find out the endpointsAdjustedPosition value
+  * @private
+  */
+  _getEndpointsAdjustedPosition() {
+    if (this.edges.length > 0) {
+      return this.edges[0].globalOptions.arrowStrikethrough === true ? this.edges[0].globalOptions.endpointsAdjustedPosition : 0;
```

Comment:
What does this do? Are you taking the global edge options from the first edge? What sense does this make?

Reply:
I guess that the property should be placed in the edges options in connection with the arrows properties. 
Maybe I missed it but is there a better solving to grab the value?

Reply:
We have complex system of option merging. That is global options are automatically merged into each edge if it doesn't have given option of its own. There should never be the need to touch global options directly and I don't see a reason why this should be global only. Ideally this would be configurable per edge end point (something like `endPointOffset: number | { from?: number, to?: number }`).

Per node configuration is also worth discussing. Because different nodes may require different offsets due to different styling. Although different end points may also need different offsets even when connected to the same node. Per edge end point configuration would be most flexible, I think.

Reply:
Yes you are right, per edge and end point should be the best configuration. 
Thanks a lot :)